### PR TITLE
Correct "no subway route" grammar

### PR DESCRIPTION
**Pull request by verhovsky**

### CODE REVIEW

File: iphone/Maps/LocalizedStrings/en.lproj/Localizable.strings [hunk 1]
```diff
@@ -1197,7 +1197,7 @@
 
 "transit_not_found" = "Subway navigation in this region is not available yet";
 
-"dialog_pedestrian_route_is_long_header" = "Subway route is not found";
+"dialog_pedestrian_route_is_long_header" = "No subway route found";
```
Comment (biodranik) [inline iphone/Maps/LocalizedStrings/en.lproj/Localizable.strings:1200]: Thanks! This string should be fixed in https://github.com/organicmaps/organicmaps/blob/master/data/strings/strings.txt

Comment (biodranik) [reply iphone/Maps/LocalizedStrings/en.lproj/Localizable.strings:1200]: @organicmaps/translations-en-gb WDYT? `No metro route found` sounds ok too?

Comment (verhovsky) [reply iphone/Maps/LocalizedStrings/en.lproj/Localizable.strings:1200]: Some cities have subways, some have metros.

**Review by biodranik — APPROVED** [review]
Thank you very much for your contribution!

**Pull request merged**